Provide a way to build the docker image without meteor

From what I understand from the readme and the docker file, to build a sandstorm docker image one needs to install a few packages and meteor (using the install pipe bash method).

On a debian wheezy I'll like to avoid installing meteor on the docker host to build a docker image to run sandstorm. 

Right now without it I get a compile error when doing "make .docker" 

```
[snip]
capnpc: sandstorm/package.capnp
  error:  couldn't find capnp.
compile: capnp/endian-reverse-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: libseccomp/arch.c
  /ekam-provider/canonical/libseccomp/arch.c:111:7: error: use of undeclared identifier 'EM_AARCH64'
          case SCMP_ARCH_AARCH64:
               ^
  /ekam-provider/c++header/seccomp.h:128:27: note: expanded from:
  #define SCMP_ARCH_AARCH64       AUDIT_ARCH_AARCH64
                                  ^
  /ekam-provider/c++header/seccomp.h:126:29: note: expanded from:
  #define AUDIT_ARCH_AARCH64      (EM_AARCH64|__AUDIT_ARCH_64BIT|__AUDIT_ARCH_LE)
                                   ^
  /ekam-provider/canonical/libseccomp/arch.c:268:7: error: use of undeclared identifier 'EM_AARCH64'
          case SCMP_ARCH_AARCH64:
               ^
  /ekam-provider/c++header/seccomp.h:128:27: note: expanded from:
  #define SCMP_ARCH_AARCH64       AUDIT_ARCH_AARCH64
                                  ^
  /ekam-provider/c++header/seccomp.h:126:29: note: expanded from:
  #define AUDIT_ARCH_AARCH64      (EM_AARCH64|__AUDIT_ARCH_64BIT|__AUDIT_ARCH_LE)
                                   ^
  /ekam-provider/canonical/libseccomp/arch.c:305:7: error: use of undeclared identifier 'EM_AARCH64'
          case SCMP_ARCH_AARCH64:
               ^
  /ekam-provider/c++header/seccomp.h:128:27: note: expanded from:
  #define SCMP_ARCH_AARCH64       AUDIT_ARCH_AARCH64
                                  ^
  /ekam-provider/c++header/seccomp.h:126:29: note: expanded from:
  #define AUDIT_ARCH_AARCH64      (EM_AARCH64|__AUDIT_ARCH_64BIT|__AUDIT_ARCH_LE)
                                   ^
  3 errors generated.
compile: joyent-http/http_parser.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/message.c++


[snip]
compile: kj/std/iostream-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/io-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/tuple-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/common.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/refcount.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/serialize-async-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/threadlocal-pthread-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/string-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/debug-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/mutex-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/mutex.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/async-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/thread.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/schema.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/capability.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/schema-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/blob.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/rpc-twoparty.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/dynamic-capability.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/fuzz-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/schema-loader.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/rpc.capnp.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/serialize-async.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/stringify-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/schema-parser-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/arena.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/layout-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/serialize.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/encoding-test.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/stringify.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/compiler/grammar.capnp.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: kj/string.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/compiler/error-reporter.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/compiler/lexer.c++
  error: invalid value 'c++1y' in '-std=c++1y'
compile: capnp/rpc-twoparty.capnp.c++
  error: invalid value 'c++1y' in '-std=c++1y'
capnpc: capnp/test.capnp
  error:  couldn't find capnp.
capnpc: capnp/test-import2.capnp
  error:  couldn't find capnp.
capnpc: capnp/test-import.capnp
  error:  couldn't find capnp.
compile: capnp/ez-rpc.c++
  error: invalid value 'c++1y' in '-std=c++1y'
==== build failed. You might need to: make update-deps ====
make: *** [tmp/.ekam-run] Error 1

```

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,11 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# This Dockerfile is for general Sandstorm usage. Building this image requires
-# a bundle to be in the current directory named 'sandstorm-0.tar.xz'. The
-# easiest way to build this image is to just run `make .docker`. This will
-# compile the Sandstorm bundle first, and then build the docker image and
-# name it `sandstorm`.
+# This Dockerfile is for general Sandstorm usage. It will fetch the latest binary distrubition of
+# Sandstorm and install it inside the container. You can build this image with `make .docker` or
+# `docker build -t sandstorm .`.
 #
 # You can then run the docker container with:
 # docker run -p 6080:6080 -i -t sandstorm
@@ -35,7 +33,7 @@ FROM ubuntu:14.04
 
 # Install sandstorm dependencies
 RUN apt-get update
-RUN apt-get install -y xz-utils
+RUN apt-get install -y curl xz-utils
 
 RUN adduser --disabled-password --gecos "" sandstorm
 USER sandstorm
@@ -43,9 +41,8 @@ ENV HOME /home/sandstorm
 ENV USER sandstorm
 
 ADD ./install.sh /install.sh
-COPY ./sandstorm-0.tar.xz /sandstorm-0.tar.xz
 
-RUN /install.sh -d -u /sandstorm-0.tar.xz
+RUN /install.sh -d -u
 
 RUN echo 'SERVER_USER=sandstorm\n\
 PORT=6080\n\

--- a/Makefile
+++ b/Makefile
@@ -180,7 +180,7 @@ sandstorm-$(BUILD)-fast.tar.xz: bundle
 	@$(call color,compress fast bundle)
 	@tar c --transform="s,^bundle,sandstorm-$(BUILD)," bundle | xz -c -0 > sandstorm-$(BUILD)-fast.tar.xz
 
-.docker: sandstorm-$(BUILD).tar.xz Dockerfile
+.docker: Dockerfile
 	@$(call color,docker build)
 	@docker build -t sandstorm .
 	@touch .docker

